### PR TITLE
Fix `clean-conversation-sidebar` for triage access collaborator

### LINKED ISSUE #4238: `clean-conversation-sidebar` is broken
Extension enabled:
Screenshot_20210414_080246
Extension disabled:
Screenshot_20210414_080311

Extension version: 21.4.13
FF version: 78.9.0esr

Note: Sorry if I am missing something here :sweat_smile:

### PR DESCRIPTION
**Pull request by yakov116**
Fixes #4238

## Test URLs
I tested on https://github.com/download-directory/download-directory.github.io/pull/37

## Screenshot
image

### CODE REVIEW

**Comment by yakov116** [comment]
Hmm this breaks other collaborator types

**Comment by yakov116** [comment]
OK from my finding, they have a unique DOM. Its a regular user with some added select dropdowns.

File: source/features/clean-conversation-sidebar.tsx [hunk 1]
```diff
@@ -101,6 +101,7 @@ async function clean(): Promise<void> {
 
 	// Linked issues/PRs
 	select('[aria-label="Link issues"] p')!.remove(); // "Successfully merging a pull request may close this issue."
+	select('[aria-label="Link issues"] p')?.remove(); // "Successfully merging a pull request may close this issue." Not shown if issues are disabled.
```
Comment (fregante) [inline source/features/clean-conversation-sidebar.tsx:104]: There are 2? The use a loop

Comment (yakov116) [reply source/features/clean-conversation-sidebar.tsx:104]: No This issue is really unrelated. I just threw it into the PR.

I can revert if you want

Comment (yakov116) [reply source/features/clean-conversation-sidebar.tsx]: Reverted

**Comment by fregante** [comment]
I think the issue here is specifically because the “Assign yourself” button is missing from the Triager’s sidebar and therefore the “children count” is zero. Since the next condition selects this  element, we can use *that* instead of this additional select. This also allows for a slight improvement of the whole if/else

**Comment by yakov116** [comment]
I lost you here

**Comment by yakov116** [comment]
I am going to close this as this feature needs a to be fixed anyways

File: source/features/clean-conversation-sidebar.tsx [hunk 2]
```diff
@@ -66,7 +66,8 @@ async function clean(): Promise<void> {
 
 	// Assignees
 	const assignees = select('.js-issue-assignees')!;
-	if (assignees.children.length === 0) {
+	// `#assignees-select-menu` check is for collaborators with only "triage access" #4238
+	if (assignees.children.length === 0 && !select('#assignees-select-menu')) {
```
Comment (fregante) [inline source/features/clean-conversation-sidebar.tsx:70]: What I meant in my previous comment is that `#assignees-select-menu` covers both cases, so `assignees.children.length === 0` is a redundant check.

The same exact thing happened in https://github.com/sindresorhus/refined-github/pull/4336#discussion_r628903336

Here's a comparison for history purposes:

## Full access

Screen Shot 1

## Triage access

114912044-8a6ad800-9ded-11eb-8540-fdef5d868a88

**Pull request closed without merging**